NoClassDefFoundError using parquet-cli  shaded jar (building with mvn -Plocal)

### Describe the bug, including details regarding any error messages, version, and platform.

building parquet-cli shaded jar, and running, I get NoClassDefFoundError on class thrift

```
$ java -jar target/parquet-cli-1.15.0-SNAPSHOT-runtime.jar help
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/thrift/TBase
        at org.apache.parquet.cli.Main.<init>(Main.java:104)
        at org.apache.parquet.cli.Main.main(Main.java:191)
Caused by: java.lang.ClassNotFoundException: org.apache.thrift.TBase
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
        ... 2 more
```

by patching to add this to parquet-cli/pom.xml, it fixes the problem
```
    <dependency>
      <groupId>org.apache.thrift</groupId>
      <artifactId>libthrift</artifactId>
      <version>${format.thrift.version}</version>
      <scope>${deps.scope}</scope>
    </dependency>
```



### Component(s)

_No response_

--- a/parquet-cli/pom.xml
+++ b/parquet-cli/pom.xml
@@ -191,6 +191,12 @@
       <scope>${deps.scope}</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.thrift</groupId>
+      <artifactId>libthrift</artifactId>
+      <version>${format.thrift.version}</version>
+      <scope>${deps.scope}</scope>
+    </dependency>
+    <dependency>
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
       <version>${jsr305.version}</version>

